regolith/schemas.py

```diff
@@ -1164,6 +1175,21 @@
             "required": False,
             "type": "string",
         },
+        'updated': {
+            "description": "date and time when the entry was created/updated",
+            "required": False,
+            "anyof_type": ["string", "datetime", "date"],
+        },
+        'year': {
```

Comment:
please could you also add a 'date' field that duplicates the year/month/date info, but allows us to input an ISO date if we want.  I am moving to ISO dates in everything so this is a good chance to update things in the contacts collection

Reply:
hi @sbillinge, I modified it. Can you check if is that what you meant?

Reply:
yes, good